scripts/run-alphanet-parachain.sh

```diff
@@ -77,16 +77,22 @@ $PARACHAIN_BINARY \
   --rpc-port $((PARACHAIN_PORT + 10 + 1)) \
   --ws-port $((PARACHAIN_PORT + 10 + 2)) \
   --validator \
+  --rpc-cors all \
+  --rpc-methods=unsafe \
+  --execution wasm \
+  --wasm-execution compiled \
   --name parachain_$PARACHAIN_INDEX \
   $PARACHAIN_BASE_PATH \
-  '-linfo,evm=trace,ethereum=trace,rpc=trace' \
-  --chain $PARACHAIN_SPEC_PLAIN  \
+  '-linfo,evm=debug,ethereum=trace,rpc=trace,cumulus_collator=debug,txpool=debug' \
+  --alice  \
+  --chain staging \
+  --account-id 6Be02d1d3665660d22FF9624b7BE0551ee1Ac91b \
```

Comment:
I believe this should be set automatically when providing alice, bob or ...

Reply:
Setting the author identity for `--alice`, `--bob`, `--dev`, etc is the desired behavior for sure. But it is not expected to happen right now. It isn't implemented yet, so for now you have to pass the address manually.

Some context:
https://github.com/PureStake/moonbeam/pull/173#discussion_r557419266
https://github.com/PureStake/moonbeam/pull/173#discussion_r558296648